CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   </summary>
 
 ### Minor
+- Update Badge component to match new specs with solid background and white text (#839)
```

Comment:
Can we use the standard way of adding entries into CHANGELOG? `ComponentName: Description (#839)`

```
Badge: Update to solid background and white text (#839)
```

Reply:
done